tests/App/config/security_v4.yaml

```diff
@@ -9,14 +9,15 @@ security:
   firewalls:
     login_area:
       pattern: ^/(login$|connect|login_hwi)
-      anonymous: true
+      anonymous: lazy
```

Comment:
Shouldn't we now have two tests? With `lazy` & `true`?

Reply:
I think we don't need it.
But I assumed that if setting anonymous to lazy does not break old tests so that would be acceptable.

Reply:
Firewall Refresh Token Listener will be enabled only if option `refresh_on_expire` set to `true` for at least one resource owner.